src/views/playlists/Playlists.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <VContainer fluid>
+    <VDataIterator
+      :footer-props="{
+        disableItemsPerPage: true,
+        itemsPerPageOptions: [numberOfItems],
+        showFirstLastPage: true,
+      }"
+      :items="filteredItems"
+      :items-per-page="numberOfItems"
+      :page.sync="pagination.page"
+    >
+      <template v-slot:header>
+        <VRow class="mb-2" justify="end">
+          <VCol lg="4" md="6" sm="8" xl="2" cols="12">
+            <VTextField
+              :label="$t('common.search')"
+              hide-details
+              prepend-inner-icon="mdi-magnify"
+              single-line
+              v-if="playlists.length > numberOfItems"
+              v-model="search"
+            />
+          </VCol>
+          <VBtn :to="{ name: 'new-playlist' }" color="success" class="ma-2">
+            <VIcon left>mdi-plus</VIcon>
+            {{ $t("music.playlist.new") }}
+          </VBtn>
+        </VRow>
+      </template>
+      <template v-slot:default="props">
+        <VRow>
+          <VCol
+            :key="item.id"
+            lg="3"
+            md="4"
+            sm="6"
+            v-for="item in props.items"
+            xl="2"
+            cols="6"
+          >
+            <VCard :to="{ name: 'playlist', params: { id: item.id } }">
+              <VCardTitle>
```

Comment:
Can we add an icon here showing the type?

Reply:
Tried this - but I didn't like it

<img width="929" alt="Screenshot 2022-06-25 at 12 15 05" src="https://user-images.githubusercontent.com/48474670/175769226-459db49b-d191-4060-a23f-5253e4a4b526.png">

Reply:
It's also kind of double information - since we already have `0 albums` written in the card